browser/importer/brave_external_process_importer_host.h

```diff
@@ -19,21 +23,35 @@ class BraveExternalProcessImporterHost : public ExternalProcessImporterHost {
       delete;
   BraveExternalProcessImporterHost& operator=(
       const BraveExternalProcessImporterHost&) = delete;
+  using MockedInstallCallback =
+      base::RepeatingCallback<void(const std::string&)>;
+  void DoNotLaunchImportForTesting();
```

Comment:
I think this method could go into `private`.

Reply:
done